include/baresip.h

```diff
@@ -825,6 +825,7 @@ enum ua_event {
 	UA_EVENT_CALL_REMOTE_SDP,     /**< param: offer or answer */
 	UA_EVENT_REFER,
 	UA_EVENT_MODULE,
+	UA_EVENT_END_OF_FILE,
```

Comment:
Is the event related to audio, video or both ?

Perhaps add some info in the event name about what it is related to ...?

Reply:
There is a `vidsrc_error_handler()` in video.c but no callback in call.c. For now only audio files can be handled. Video files could be added in a future PR.
I suggest to add Doxygen which mentions only audio files. This can be replaced if someone adds video file support.

Reply:
Maybe its better to put the video stuff in a separate PR. Normally we merge with "squash and merge", then it becomes one commit. Also its easier to review if the PR is small.